dashboard/src/actions/runtimes.ts

```diff
@@ -0,0 +1,22 @@
+import { Dispatch } from "redux";
```

Comment:
I think these actions should really be part of functions and the function state (e.g. the catalog actions/state holds all the state for brokers, instances, plans, etc.), unless there's any particular reason you wanted this to be separate?

Reply:
I agree they're different entities, but they are part of the same component so I think it makes sense to group them - especially since that's what we're doing with Service Catalog (brokers, instances, plans are all separate entities).